app_server/README.md

```diff
@@ -1,9 +1,29 @@
 [![Build Status][travis-image]][travis-url] [![Coverage Status][coveralls-image]][coveralls-url]
 
+## Setting up the Database
+As WorldScope relies on MySQL, it is required to set up a MySQL database. Download a copy from http://dev.mysql.com/downloads/ based on your Operating System. It will be good to keep MySQL version at least v5.5
+
+* Note the MySQL username and password 
+* Create database named `'worldscope_db'` or other names but do update the `name` field in the next step
+```
+mysql> CREATE DATABASE worldscope_db;
+```
+* Edit the `name`, `username` and `password` fields accordingly in configuration file located in `worldscope/app_server/config/DatabaseConfig.js`
+
```

Comment:
Can provide a relative link to a file in the repository itself using this syntax for example:

... accordingly in the  [database configuration file](config/DatabaseConfig.js)